### PR TITLE
feat(Buttons, Touchable, Menu): accessibility improvements

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1561)

### CODE REVIEW

**Review by pladaria — APPROVED** [review]

File: src/button.tsx [hunk 1]
```diff
@@ -354,8 +356,11 @@ const Button = React.forwardRef<TouchableElement, ButtonProps & {type: ButtonTyp
         style: {cursor: props.fake ? 'pointer' : undefined, ...props.style},
         trackingEvent: props.trackingEvent ?? (props.trackEvent ? createDefaultTrackingEvent() : undefined),
         dataAttributes: props.dataAttributes,
+        'aria-label': props['aria-label'],
         'aria-controls': props['aria-controls'],
         'aria-expanded': props['aria-expanded'],
+        'aria-haspopup': props['aria-haspopup'],
+        'aria-busy': showSpinner ? true : undefined,
```
Comment (yceballost) [inline src/button.tsx:363]: the current implementation is already working with voice-over while loading state.. so... I wonder why do we need aria-busy 🤔

**Review by yceballost — APPROVED** [review]

File: src/button.tsx (hunk 1 above)
Comment (atabel) [reply src/button.tsx:363]: If it has no effect, I'd remove it

File: src/__stories__/sheet-story.tsx [hunk 2]
```diff
@@ -42,6 +42,8 @@ export const Default: StoryComponent = () => {
             <Stack space={16}>
                 <ButtonPrimary
                     disabled={open}
+                    aria-expanded={open}
```
Comment (atabel) [inline src/__stories__/sheet-story.tsx:45]: you may want to change playroom sheet snippets too

File: src/button.tsx (hunk 1 above)
Comment (aweell) [reply src/button.tsx:363]: Agree. Removed

Comment (marcoskolodny) [reply src/button.tsx]: Good catch, I tested a button with aria-busy to see if it was working, but I didn't notice it was the same behavior as the case when it doesn't have aria-busy.

Removed this prop :)

File: src/__stories__/sheet-story.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/__stories__/sheet-story.tsx:45]: done :)

**Review by atabel — APPROVED** [review]

**Review by aweell — APPROVED** [review]

**Pull request merged**